modules/codemod/lib/v9/spec/deprecateStack.spec.ts

```diff
@@ -317,7 +317,7 @@ describe('Canvas Kit Deprecate Stack Codemod', () => {
       expectTransform(input, expected);
     });
 
-    it('should properly transform the spacing prop on specific components that extend Stack', () => {
+    it('should properly transform the spacing prop on the Breadcrumbs component that extends Stack', () => {
```

Comment:
Maybe I'm missing something, but I'm confused about this test. As far as I can tell, there are no Breadcrumbs subcomponents that extend `Stack` in v8. I see in the [v8 Breadcrumbs docs](https://workday.github.io/canvas-kit/?path=/docs/components-navigation-breadcrumbs--basic) that `Breadcrumbs.List` and `Breadcrumbs.Item` extend `Flex`, but I don't see any subcomponents that extend `Stack`.

If this is true, then the codemod shouldn't be touching canvas-kit Breadcrumbs at all (and this test doesn't need to exist).

Reply:
I have removed breadcrumbs from the array for checking those imports